hw4_string.py

```diff
@@ -0,0 +1,59 @@
+# Задача 1 - Написати програму, в якій необхідно видалити всі вказані літери з стрінги(string)
+line_1 = input("Enter text: ")
+letter_need_to_delete = input("Enter letter: ")
+final_text_1 = line_1.replace(letter_need_to_delete,"")
+print(f'Result is {final_text_1}')
+
+# Написати програму, яка візьме string і в кожному слові замінить першу літеру на велику.
+
+line_2 = input("Enter text: ")
+final_text_2 = str.title(line_2)
+print(f' Final text is {final_text_2}')
+
+#Напишіть програму яка перевертає стрінгу
+
+line_3 = input('Enter text for changing: ')
+final_text_3 = line_3[::-1]
+print(f'Changing text is {final_text_3}')
+
+# Напишіть програму яка приймає на вхід 2 стрінги та порівнює їх
+
+line_4 = input('Enter text: ')
+line_4_compare = input('Enter text for compare :')
+if line_4 == line_4_compare:
+    print("Texts are equals")
+else:
+    print("Texts are not equals")
```

Comment:
This can be simplier
first = input("Enter first string: ")
second = input("Enter second string: ")

print(f"Is string {first} and string {second} the same? - {first == second}")